src/components/SubscriptionSwitchesList/SubscriptionSwitchesList.js

```diff
@@ -105,10 +106,15 @@ const SubscriptionSwitchesList = ({
       )
     : [];
 
+  // Filter out the pause subscription
+  const availableFiltered =
+    Array.isArray(availableSorted) &&
```

Comment:
is there really an option that it won't be an array? Can't we just check if it exists `availableSorted?.filter... `?